2017-06-22T12:11:09.492054+00:00 app[web.1]: 
2017-06-22T12:11:09.492056+00:00 app[web.1]: events.js:160
2017-06-22T12:11:09.492058+00:00 app[web.1]:       throw er; // Unhandled 'error' event
2017-06-22T12:11:09.492058+00:00 app[web.1]:       ^
2017-06-22T12:11:09.493196+00:00 app[web.1]: MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]
2017-06-22T12:11:09.493198+00:00 app[web.1]:     at Pool.<anonymous> (/app/node_modules/mongodb-core/lib/topologies/server.js:328:35)
2017-06-22T12:11:09.493199+00:00 app[web.1]:     at emitOne (events.js:96:13)
2017-06-22T12:11:09.493200+00:00 app[web.1]:     at Pool.emit (events.js:188:7)
2017-06-22T12:11:09.493201+00:00 app[web.1]:     at Connection.<anonymous> (/app/node_modules/mongodb-core/lib/connection/pool.js:280:12)
2017-06-22T12:11:09.493201+00:00 app[web.1]:     at Connection.g (events.js:292:16)
2017-06-22T12:11:09.493202+00:00 app[web.1]:     at emitTwo (events.js:106:13)
2017-06-22T12:11:09.493203+00:00 app[web.1]:     at Connection.emit (events.js:191:7)
2017-06-22T12:11:09.493203+00:00 app[web.1]:     at Socket.<anonymous> (/app/node_modules/mongodb-core/lib/connection/connection.js:177:49)
2017-06-22T12:11:09.493204+00:00 app[web.1]:     at Socket.g (events.js:292:16)
2017-06-22T12:11:09.493205+00:00 app[web.1]:     at emitOne (events.js:96:13)
2017-06-22T12:11:09.493205+00:00 app[web.1]:     at Socket.emit (events.js:188:7)
2017-06-22T12:11:09.493206+00:00 app[web.1]:     at emitErrorNT (net.js:1277:8)
2017-06-22T12:11:09.493207+00:00 app[web.1]:     at _combinedTickCallback (internal/process/next_tick.js:80:11)
2017-06-22T12:11:09.493208+00:00 app[web.1]:     at process._tickCallback (internal/process/next_tick.js:104:9)
2017-06-22T12:11:09.552191+00:00 app[web.1]: 
2017-06-22T12:11:09.564804+00:00 app[web.1]: npm ERR! Linux 3.13.0-121-generic
2017-06-22T12:11:09.565169+00:00 app[web.1]: npm ERR! argv "/app/.heroku/node/bin/node" "/app/.heroku/node/bin/npm" "start"
2017-06-22T12:11:09.565435+00:00 app[web.1]: npm ERR! node v6.11.0
2017-06-22T12:11:09.578923+00:00 app[web.1]: npm ERR! npm  v3.10.10
2017-06-22T12:11:09.579203+00:00 app[web.1]: npm ERR! code ELIFECYCLE
2017-06-22T12:11:09.579407+00:00 app[web.1]: npm ERR! mytodoapp@1.0.0 start: `node app`
2017-06-22T12:11:09.579590+00:00 app[web.1]: npm ERR! Exit status 1
2017-06-22T12:11:09.579787+00:00 app[web.1]: npm ERR! 
2017-06-22T12:11:09.579980+00:00 app[web.1]: npm ERR! Failed at the mytodoapp@1.0.0 start script 'node app'.
2017-06-22T12:11:09.580156+00:00 app[web.1]: npm ERR! Make sure you have the latest version of node.js and npm installed.
2017-06-22T12:11:09.580320+00:00 app[web.1]: npm ERR! If you do, this is most likely a problem with the mytodoapp package,
2017-06-22T12:11:09.580495+00:00 app[web.1]: npm ERR! not with npm itself.
2017-06-22T12:11:09.591762+00:00 app[web.1]: npm ERR! Tell the author that this fails on your system:
2017-06-22T12:11:09.591889+00:00 app[web.1]: npm ERR!     node app
2017-06-22T12:11:09.592027+00:00 app[web.1]: npm ERR! You can get information on how to open an issue for this project with:
2017-06-22T12:11:09.592160+00:00 app[web.1]: npm ERR!     npm bugs mytodoapp
2017-06-22T12:11:09.592299+00:00 app[web.1]: npm ERR! Or if that isn't available, you can get their info via:
2017-06-22T12:11:09.592640+00:00 app[web.1]: npm ERR!     npm owner ls mytodoapp
2017-06-22T12:11:09.592747+00:00 app[web.1]: npm ERR! There is likely additional logging output above.
2017-06-22T12:11:09.609477+00:00 app[web.1]: 
2017-06-22T12:11:09.609778+00:00 app[web.1]: npm ERR! Please include the following file with any support request:
2017-06-22T12:11:09.609927+00:00 app[web.1]: npm ERR!     /app/npm-debug.log
2017-06-22T12:11:09.757167+00:00 heroku[web.1]: State changed from starting to crashed
2017-06-22T12:11:09.741663+00:00 heroku[web.1]: Process exited with status 1
2017-06-22T12:11:19.205826+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/" host=fathomless-ocean-89963.herokuapp.com request_id=204045b2-45c7-403e-b240-1d6adb5e0dab fwd="1.32.39.141" dyno= connect= service= status=503 bytes= protocol=https
2017-06-22T12:11:20.773301+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/favicon.ico" host=fathomless-ocean-89963.herokuapp.com request_id=e5f2c9ed-19f2-4d9a-a544-ceb561053a3f fwd="1.32.39.141" dyno= connect= service= status=503 bytes= protocol=https
2017-06-22T12:28:52.047914+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/" host=fathomless-ocean-89963.herokuapp.com request_id=608648d4-94b6-422a-b8b5-ce6a5bf11dc2 fwd="1.32.39.141" dyno= connect= service= status=503 bytes= protocol=https
2017-06-22T12:28:53.246183+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/favicon.ico" host=fathomless-ocean-89963.herokuapp.com request_id=071c249c-5b11-4024-83dd-9a9778c027c3 fwd="1.32.39.141" dyno= connect= service= status=503 bytes= protocol=https
2017-06-22T12:37:47.193985+00:00 heroku[web.1]: State changed from crashed to starting
2017-06-22T12:37:49.477272+00:00 heroku[web.1]: Starting process with command `npm start`
2017-06-22T12:37:52.832826+00:00 app[web.1]: 
2017-06-22T12:37:52.832841+00:00 app[web.1]: > mytodoapp@1.0.0 start /app
2017-06-22T12:37:52.832842+00:00 app[web.1]: > node app
2017-06-22T12:37:52.832843+00:00 app[web.1]: 
2017-06-22T12:37:53.745061+00:00 app[web.1]: Server Running on port 5000
2017-06-22T12:37:53.761273+00:00 app[web.1]:       throw er; // Unhandled 'error' event
2017-06-22T12:37:53.761269+00:00 app[web.1]: 
2017-06-22T12:37:53.761272+00:00 app[web.1]: events.js:160
2017-06-22T12:37:53.762933+00:00 app[web.1]:     at Pool.<anonymous> (/app/node_modules/mongodb-core/lib/topologies/server.js:328:35)
2017-06-22T12:37:53.761273+00:00 app[web.1]:       ^
2017-06-22T12:37:53.762931+00:00 app[web.1]: MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connect ECONNREFUSED 127.0.0.1:27017]
2017-06-22T12:37:53.762936+00:00 app[web.1]:     at Connection.<anonymous> (/app/node_modules/mongodb-core/lib/connection/pool.js:280:12)
2017-06-22T12:37:53.762934+00:00 app[web.1]:     at emitOne (events.js:96:13)
2017-06-22T12:37:53.762935+00:00 app[web.1]:     at Pool.emit (events.js:188:7)
2017-06-22T12:37:53.762938+00:00 app[web.1]:     at Socket.<anonymous> (/app/node_modules/mongodb-core/lib/connection/connection.js:177:49)
2017-06-22T12:37:53.762936+00:00 app[web.1]:     at Connection.g (events.js:292:16)
2017-06-22T12:37:53.762937+00:00 app[web.1]:     at emitTwo (events.js:106:13)
2017-06-22T12:37:53.762937+00:00 app[web.1]:     at Connection.emit (events.js:191:7)
2017-06-22T12:37:53.762940+00:00 app[web.1]:     at Socket.emit (events.js:188:7)
2017-06-22T12:37:53.762939+00:00 app[web.1]:     at Socket.g (events.js:292:16)
2017-06-22T12:37:53.762939+00:00 app[web.1]:     at emitOne (events.js:96:13)
2017-06-22T12:37:53.762942+00:00 app[web.1]:     at process._tickCallback (internal/process/next_tick.js:104:9)
2017-06-22T12:37:53.762942+00:00 app[web.1]:     at _combinedTickCallback (internal/process/next_tick.js:80:11)
2017-06-22T12:37:53.762941+00:00 app[web.1]:     at emitErrorNT (net.js:1277:8)
2017-06-22T12:37:53.817174+00:00 app[web.1]: npm ERR! node v6.11.0
2017-06-22T12:37:53.793844+00:00 app[web.1]: 
2017-06-22T12:37:53.816941+00:00 app[web.1]: npm ERR! argv "/app/.heroku/node/bin/node" "/app/.heroku/node/bin/npm" "start"
2017-06-22T12:37:53.817657+00:00 app[web.1]: npm ERR! mytodoapp@1.0.0 start: `node app`
2017-06-22T12:37:53.812605+00:00 app[web.1]: npm ERR! Linux 3.13.0-121-generic
2017-06-22T12:37:53.817356+00:00 app[web.1]: npm ERR! npm  v3.10.10
2017-06-22T12:37:53.817532+00:00 app[web.1]: npm ERR! code ELIFECYCLE
2017-06-22T12:37:53.817777+00:00 app[web.1]: npm ERR! Exit status 1
2017-06-22T12:37:53.817952+00:00 app[web.1]: npm ERR! 
2017-06-22T12:37:53.818054+00:00 app[web.1]: npm ERR! Failed at the mytodoapp@1.0.0 start script 'node app'.
2017-06-22T12:37:53.818729+00:00 app[web.1]: npm ERR! Tell the author that this fails on your system:
2017-06-22T12:37:53.818165+00:00 app[web.1]: npm ERR! Make sure you have the latest version of node.js and npm installed.
2017-06-22T12:37:53.818424+00:00 app[web.1]: npm ERR! not with npm itself.
2017-06-22T12:37:53.818305+00:00 app[web.1]: npm ERR! If you do, this is most likely a problem with the mytodoapp package,
2017-06-22T12:37:53.819284+00:00 app[web.1]: npm ERR!     npm bugs mytodoapp
2017-06-22T12:37:53.819915+00:00 app[web.1]: npm ERR! There is likely additional logging output above.
2017-06-22T12:37:53.819065+00:00 app[web.1]: npm ERR! You can get information on how to open an issue for this project with:
2017-06-22T12:37:53.818852+00:00 app[web.1]: npm ERR!     node app
2017-06-22T12:37:53.819791+00:00 app[web.1]: npm ERR!     npm owner ls mytodoapp
2017-06-22T12:37:53.819401+00:00 app[web.1]: npm ERR! Or if that isn't available, you can get their info via:
2017-06-22T12:37:53.833305+00:00 app[web.1]: 
2017-06-22T12:37:53.833542+00:00 app[web.1]: npm ERR! Please include the following file with any support request:
2017-06-22T12:37:53.833666+00:00 app[web.1]: npm ERR!     /app/npm-debug.log
2017-06-22T12:37:53.967104+00:00 heroku[web.1]: State changed from starting to crashed
2017-06-22T12:37:53.952362+00:00 heroku[web.1]: Process exited with status 1
2017-06-22T12:44:49.959520+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/" host=fathomless-ocean-89963.herokuapp.com request_id=2a0f198d-4c3d-4e05-bf26-fc76e884f98d fwd="1.32.39.141" dyno= connect= service= status=503 bytes= protocol=https
2017-06-22T12:44:51.506807+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/favicon.ico" host=fathomless-ocean-89963.herokuapp.com request_id=7dbaf6ce-b863-4b07-a984-3e3b3d00c9c3 fwd="1.32.39.141" dyno= connect= service= status=503 bytes= protocol=https
